dev/image/Dockerfile

```diff
@@ -51,23 +48,20 @@ RUN curl -fsSL https://github.com/kubernetes-sigs/kubebuilder/releases/download/
 ### MySQL client ###
 RUN install-packages mysql-client
 
-# golangci-lint
-RUN cd /usr/local && curl -fsSL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s v1.42.0
```

Comment:
golangci comes with the [base image ](https://github.com/gitpod-io/workspace-images/blob/ab4d385b767547dc10fe0f3ed5d8a3435b8a5e9e/chunks/lang-go/Dockerfile#L25)